migrating_from_ocp_3_to_4/premigration-checklists-3-4.adoc

```diff
@@ -7,15 +7,24 @@ toc::[]
 
 Before you migrate your application workloads with the {mtc-full} ({mtc-short}), review the following checklists.
 
-[id="source-cluster-checklist_{context}"]
-== Source cluster checklist
+[id="resources_{context}"]
+== Resources
+
+* [ ] If your application uses an internal service network or an external route for communicating with services, the relevant route exists.
```

Comment:
Add an action like "check that" or "verify that". A declarative statement ("x exists") is not an action item. GLOBAL